<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="Ports">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchCOM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Keithley 2410</string>
    </property>
    <widget class="QComboBox" name="Compliance_Unit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>240</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>nA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>µA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Compliance</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="IV_step">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.050000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Voltage</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Voltage</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Measurement Delay</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="IV_meanbase">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Current</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Compliance">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
     <property name="value">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="Connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltage Step</string>
     </property>
    </widget>
    <widget class="QComboBox" name="IV_imax_unit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>nA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>µA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="IV_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IVCurve</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Set_Compl">
     <property name="geometry">
      <rect>
       <x>-80</x>
       <y>240</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Compliance</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="IV_end">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1100.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mean Base</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Close">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="IV_start">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-1100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1100.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="IV_delay">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>210</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>3000</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="IV_imax">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
     <property name="value">
      <double>200.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="test">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>test</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Temperature Board</string>
    </property>
    <widget class="QPushButton" name="Connect_TB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Close_TB">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto-Measure</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sensor Number:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SensorChannel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>43</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdtemp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>-99.900000000000006</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>130</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>32</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>111</height>
      </rect>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>350</number>
     </property>
     <property name="value">
      <number>180</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="Measure">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Measure</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Cal_sensors">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>240</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="Cal_temp_is">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-60.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="Cal_filename">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>270</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Calibration.dat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Used Sensors:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cal_addpoint">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Point</string>
     </property>
    </widget>
    <widget class="QPushButton" name="automeasureIVonTemp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IV on Temp</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>200</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-60.000000000000000</double>
     </property>
     <property name="maximum">
      <double>200.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Delta T</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>204</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>K</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
